src/main/java/com/rsmart/kuali/kfs/pdp/service/impl/AchBundlerFormatServiceImpl.java

```diff
@@ -90,7 +87,8 @@ public AchBundlerFormatServiceImpl() {
      */
     @Override
     public void performFormat(Integer processId) throws FormatException {
-        LOG.debug("performFormat() started - ACH Bundler Mod");
+        LOG.info("performFormat() started - ACH Bundler Mod for processId {}", processId);
+        final String pdpFormatFailureToEmailAddress = this.getAchBundlerHelperService().getPdpFormatFailureToEmailAddress();

```

Comment:
Should 'this.' be removed? I think it is unnecessary and KualiCo cleaned these up in base code.

Reply:
Removing